helloworld-mutual-ssl-secured/restore-certs.cli

```diff
@@ -0,0 +1,8 @@
+#remove the keypairs and certificates from the keystore and truststore
+/subsystem=elytron/key-store=serverTS:remove-alias(alias=example)
+/subsystem=elytron/key-store=clientKS:remove-alias(alias=example)
```

Comment:
I think the alias needs to be updated based on the values used in the cli scripts above.

Reply:
fixed!
Thanks @fjuma